Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/model/orm/SiteModel.java

```diff
@@ -63,13 +63,15 @@ public void storeConfig(SiteConfig config) {
 
     public void storeUserSettings(JsonSettings userSettings) {
         this.userSettings = gson.toJson(userSettings);
-        Logger.test("userSettings = " + this.userSettings);
```

Comment:
The userSettings in SiteModel are only used for captcha settings currently. Logins are not stored in these fields; see Chan4, specifically the no-arg constructor and the initializeSettings function. These changes aren't necessary.

Reply:
Done.